config/action.d/allow-iptables-multiport.conf

```diff
@@ -0,0 +1,59 @@
+# Fail2Ban configuration file for allowing hosts
+#
+# WARNING
+# Please be aware that all users behind NAT will access the service on the specified port.
+# You should protect this service with another jail that has very long bantime.
+
+[INCLUDES]
+
+before = iptables-common.conf
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = iptables -N f2b-<name>
+              iptables -A f2b-<name> -j <blocktype>
```

Comment:
On Sat, 11 Jul 2015, Viktor Szépe wrote:
>  > +actionstart = iptables -N f2b-<name>
>  > +              iptables -A f2b-<name> -j <blocktype>

>      add "returntype" variable defaulting to "RETURN" in -commons

>    I don't dare to touch other filters.

be bold/be brave! ;)  (now you have my blessing)

>      why not ACCEPT?
>    # Notes: ACCEPT skips other chains
>    It is necessary to be able to block "pass2allow"-ed IP-s.

ah right -- here the target/default rule is to reject... smart and nice

-- 
Yaroslav O. Halchenko, Ph.D.
http://neuro.debian.net http://www.pymvpa.org http://www.fail2ban.org
Research Scientist,            Psychological and Brain Sciences Dept.
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik